pkg/flowaggregator/flowaggregator.go

```diff
@@ -315,6 +339,8 @@ func (fa *flowAggregator) Run(stopCh <-chan struct{}) {
 		fa.configWatcher.Close()
 	}()
 	<-stopCh
+	fa.collectingProcess.Stop()
+	fa.aggregationProcess.Stop()
```

Comment:
I think these two functions, which are originally deferred statements, might be called before all child goroutines complete?

Reply:
Do you mean that we should guarantee that these 2 processes are not stopped until `fa.flowExportLoop` returns? Is this required by the implementation?

Reply:
I changed the code (see last commit) to preserve the previous behavior.

Reply:
> Is this required by the implementation?
No, I was just thinking whether calling these two processes before the completion of fa.flowExportLoop could pose any problems. But it looks like it is the same behavior as before. And we just want to ensure that all goroutines finish before the Run() function returns, regardless of whether the processes are terminated prematurely?

Reply:
I think the previous commit more preferable as two wg are providing the same functionality? But I don't have a strong preference either way.

Reply:
they don't provide the same functionality. they ensure that the IPFIX processes are stopped after the `flowExportLoop` returns, which is a change I made based on your original comment. I don't think there is harm in preserving that behavior, it feels like the safer thing to do.